package.json

```diff
@@ -17,12 +17,11 @@
     "config": "^3.0.0"
   },
   "scripts": {
-    "test": "set NODE_ENV=test&node node_modules/protractor/bin/protractor conf/docker-conf.js",
-    "production": "set NODE_ENV=production&node node_modules/protractor/bin/protractor conf.js",
-    "localtest": "set NODE_ENV=production&node node_modules/protractor/bin/protractor conf/local.conf.js",
-    "singletest": "set NODE_ENV=production&node node_modules/protractor/bin/protractor conf/single.conf.js",
-    "paralleltest": "set NODE_ENV=production&node node_modules/protractor/bin/protractor conf/parallel.conf.js",
-    "docker-test": "set NODE_ENV=production&node node_modules/protractor/bin/protractor conf/docker-conf.js"
+    "test": "set NODE_ENV=test&node node_modules/protractor/bin/protractor config/config_exec/protractor_conf/simple.conf.js",
```

Comment:
We have too many options, this is temporary once we have actual implementation we may don't need this many.